src/vivarium/framework/results/strategies.py

```diff
@@ -61,3 +61,41 @@ def __call__(self, population: pd.DataFrame) -> pd.DataFrame:
                 result_data = population[self._target].apply(self._mapper)
         result[self._mapped_column] = result_data.astype('category')
         return result
+
+
+class BinningStrategy(MappingStrategy):
+    """A strategy for expanding results data by binning continuous columns.
+
+    A binning strategy is an often-used special case of a
+    :obj:`MappingStrategy`. It maps a continuous column into a categorical
+    column by slicing up the range of the continuous column into discrete
+    bins.
+
+    """
+
+    def __init__(self, target: str, binned_column: str, bins: List[Union[int, float, pd.Timestamp]],
+                 labels: List[str], **cut_kwargs):
+        """
+        Parameters
+        ----------
+        target
+            The name of a true state table column or value pipeline to be
```

Comment:
For both MappingStrategy and BinningStrategy would "source" or "input" be more appropriate than "target"?